wikipron/cli.py

```diff
@@ -74,6 +74,16 @@ def _get_cli_args(args: List[str]) -> argparse.Namespace:
             "To disable such IPA segmentation, apply this flag."
         ),
     )
+    parser.add_argument(
```

Comment:
This flag (and throughout) should probably be called "--no-skip-spaces-word" since we use the term `word` and `pron` throughout.